src/caselawclient/models/press_summaries.py

```diff
@@ -25,16 +27,18 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super(PressSummary, self).__init__(self.document_noun, *args, **kwargs)
 
     @cached_property
-    def neutral_citation(self) -> str:
-        return self.body.get_xpath_match_string(
-            "/akn:akomaNtoso/akn:doc/akn:preface/akn:p/akn:neutralCitation/text()",
-            {
-                "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
-            },
+    def neutral_citation(self) -> NeutralCitationString:
+        return NeutralCitationString(
+            self.body.get_xpath_match_string(
+                "/akn:akomaNtoso/akn:doc/akn:preface/akn:p/akn:neutralCitation/text()",
+                {
+                    "akn": "http://docs.oasis-open.org/legaldocml/ns/akn/3.0",
+                },
+            )
         )
 
     @property
-    def best_human_identifier(self) -> str:
+    def best_human_identifier(self) -> NeutralCitationString:
```

Comment:
This should still be a `str`; the fact it's an NCN right now doesn't mean it won't be something less specific in future.
```suggestion
    def best_human_identifier(self) -> str:
```